CMakeLists.txt

```diff
@@ -36,11 +36,22 @@ common_find_package(Boost REQUIRED COMPONENTS
   filesystem system program_options unit_test_framework)
 common_find_package(vmmlib REQUIRED)
 common_find_package(OpenMP)
+common_find_package(libuv)
+
+if(libuv_FOUND)
+  if(libuv_VERSION VERSION_LESS 1.9.0)
+    add_definitions(-DUV_DISCONNECT=4)
```

Comment:
It's not defined in the ancient libuv version of Ubuntu 16.04, but the uvw C++ wrapper library needs it. I couldn't go further back for an old enough version of uvw, so that's the shortest path to make things work.